app/controllers/windows_controller.rb

```diff
@@ -0,0 +1,29 @@
+class WindowsController < ApplicationController
```

Comment:
So we are calling them Windows, not Spaces?

Reply:
Will change in next commit